### ISSUE TITLE
Dimensions intermittently lose all stored power

### ISSUE THREAD

**Issue by OldSedan** [McJty/RFToolsDimensions#99]
I'm having rftools dimensions occasionally lose all stored power on load. Seems to only happen when going to the dimension after a server restart, but it's hard to reproduce so I'm not sure about that. It's not an issue with chunk loading the power generation and dimension builder, just based on the amounts of power involved. Most recently, I was standing in a dimension stably and cheerfully, quit minecraft, restarted, and was promptly killed because the dimension was unpowered.

MC 1.10.2
forge 12.18.3.2254
compatlayer 1.10-0.2.2
mcjtylib 1.1x-2.3.9
rftools 1.1x-5.87
rftoolsdim-1.1x-4.55

I'm now realizing I'm behind on updates for mcjtylib, compatlayer, and rftools; please excuse me if this has already been fixed! If I see it again with latest versions I'll let you know.

Thanks!

**Comment by OldSedan** [comment]
Updating mods didn't prevent recurrence.

**Comment by McJty** [comment]
Can you give me your logs?

**Comment by OldSedan** [comment]
Sure thing, last time it happened was at like 5:30 yesterday but here's everything I've got in case it helps.

[logs.zip](https://github.com/McJty/RFToolsDimensions/files/920014/logs.zip)

**Comment by McJty** [comment]
Is the server exited cleanly? i.e. not with forceful interrupt or something?

**Comment by OldSedan** [comment]
Yep, exits just fine. There was one occasion where the server crashed almost immediately after loading, but that's out of a dozen or so instances. 

I should mention in case it's helpful that I've been using ftbutils chunk loading, and it's become clear that doesn't work in a way that's not straightforward. I'd ruled it out as a cause because I figured that a few minutes with the power gen unloaded wouldn't be enough to drain like 40mil stored RF but now I'm not so sure; there's been weird behavior with enderio conduits losing connections that makes me think something could be getting interrupted. I grabbed chickenchunks this morning and turned ftbutils chunk loading off, so again I'll report back if/when I get the dimensions problem again.

Thanks man!
